tests/utils/test_db.py

```diff
@@ -0,0 +1,175 @@
+from unittest.mock import patch
+
+from page_analyzer.utils.db import (
+    create_check,
+    create_url,
+    get_url_by_name,
+    get_url_checks,
+    get_url_data,
+    get_urls,
+)
+
+
+def test_create_url(test_database: None):
+    # given
+    url_name = "https://example.com"
+
+    # when
+    url_id = create_url(url_name)
+
+    # then
+    created_url = get_url_data(url_id)
+    assert created_url["name"] == url_name
+    assert url_id == created_url["id"]
```

Comment:
let's use a consistent asserts structure, like `actual` on the left and `expected` on the right

```suggestion
    assert created_url["name"] == url_name
    assert created_url["id"] == url_id
```